wca-regulations.md

```diff
@@ -458,10 +458,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 ## <article-C><one-handed><onehandedsolving> Article C: One-Handed Solving
 
 - C1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
-    - C1b) During the solve, the competitor must use only one hand to touch the puzzle. Penalty: disqualification of the attempt (DNF).
+    - C1b) During the solve, the competitor must use only one hand to manipulate the puzzle. Penalty: disqualification of the attempt (DNF).
```

Comment:
I strongly prefer "operate" to "manipulate" (as in [9a](https://github.com/thewca/wca-regulations/commit/2a8bb4bab952cecad30f4a74d5d0e90defdb7930). How would you feel about using that throughout this PR?

Reply:
As @kingmathyall mentioned in the related issue, "manipulate" is also used in other rules. So I think we should either change all occurrences or leave it as it is.